src/locale/zh_Hant/LC_MESSAGES/django.po

```diff
@@ -861,8 +861,10 @@ msgid "labels"
 msgstr ""
 
 #: proposals/models.py:272
+#, fuzzy
+#| msgid "additional speaker"
 msgid "first time speaker"
-msgstr "第一次擔任講者"
+msgstr "額外講者"
```

Comment:
```suggestion
msgid "first time speaker"
msgstr "第一次擔任講者"
```

Reply:
Revise in 045d418bdee5359283dd4245bedbd8a042197dc3

Reply:
Thank you. I revised it in 045d418